layers/+lang/elixir/packages.el

```diff
@@ -105,7 +110,28 @@
       "cf" 'alchemist-compile-file
       "c:" 'alchemist-compile
 
-      "," 'alchemist-goto-jump-back)
+      "gg" 'alchemist-goto-definition-at-point
+      "." 'alchemist-goto-definition-at-point
+      "gb" 'alchemist-goto-jump-back
+      ","  'alchemist-goto-jump-back
+      "gN" 'alchemist-goto-jump-to-previous-def-symbol
+      "gn" 'alchemist-goto-jump-to-next-def-symbol
+      "gj" 'alchemist-goto-list-symbol-definitions
+
+      "Xi" 'alchemist-hex-info-at-point
+      "Xr" 'alchemist-hex-releases-at-point
+      "XR" 'alchemist-hex-releases
+      "XI" 'alchemist-hex-info
+      "Xs" 'alchemist-hex-search
+
+      "ol" 'alchemist-macroexpand-once-current-line
+      "oL" 'alchemist-macroexpand-once-print-current-line
+      "ok" 'alchemist-macroexpand-current-line
+      "oK" 'alchemist-macroexpand-print-current-line
+      "oi" 'alchemist-macroexpand-once-region
+      "oI" 'alchemist-macroexpand-once-print-region
+      "or" 'alchemist-macroexpand-region
+      "oR" 'alchemist-macroexpand-print-region)
```

Comment:
My understanding is that the elixir macroexpand is not interactive, it is meant to be "expand once" or "expand completely" for different region sizes and with optional "printing" (appending below the current region as comments).

e.g. : 
<img width="659" alt="screen shot 2017-04-20 at 5 05 41 am" src="https://cloud.githubusercontent.com/assets/42988/25230229/143e823a-2588-11e7-858c-5d03cd66f607.png">